src/data.py

```diff
@@ -227,7 +227,14 @@ def parse_eatery_type(eatery):
 def parse_static_op_hours(hours_list, eatery_id, dining_items):
   weekdays = {}
   for hours in hours_list:
-    for weekday in WEEKDAYS[hours['weekday']]:
+    if '-' in hours['weekday']:
```

Comment:
why would there be hyphens in the hours?

Reply:
in the externalEateries file, we represent days as 'monday-wednesday' or just 'wednesday' for example.